sdk/src/main/java/com/kin/ecosystem/recovery/BackupAndRestoreImpl.java

```diff
@@ -23,7 +23,8 @@
 
 public class BackupAndRestoreImpl implements BackupAndRestore {
 
-	protected final BackupManager backupManager;
+	private final Activity activity;
+	protected BackupManager backupManager;
```

Comment:
can we change to private?

Reply:
@drEfrati ^^^

Reply:
I didn't change it, it was protected before

Reply:
I just removed the final